### DISCUSSION TITLE
Failed to install the package

### DISCUSSION THREAD

**Discussion by adeluna84** [spatie/laravel-sitemap#349] [Q&A]
Im using Laravel 7 and i cant install sitemap.

![image](https://user-images.githubusercontent.com/46869840/101431862-56c2f300-38cd-11eb-8247-35fe88e5b60f.png)

**Comment by cosmin84** [comment]
Same here. Tried to install the package today and got the same error.

My setup:
- Laravel Framework 7.30.0
- PHP 7.4.13
- Composer version 2.0.8
